week9/answers.sql

```diff
@@ -1,3 +1,193 @@
 -- Create your tables, views, functions and procedures here!
 CREATE SCHEMA social;
 USE social;
+
+CREATE TABLE users (
+	user_id INT UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT,
+	first_name VARCHAR(30) NOT NULL,
+	last_name VARCHAR(30) NOT NULL,
+	email TEXT NOT NULL,
+	-- Found current timestamp here:
+	-- https://www.w3schools.com/sql/func_sqlserver_current_timestamp.asp
+	created_on DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE sessions (
+  session_id INT UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  user_id INT UNSIGNED NOT NULL,
+  created_on DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_on DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
Checkout the Week 11 & 12: Dates and Time lesson. It covers why you would want to use a TIMESTAMP here. It also covers the "ON UPDATE" clause you can use without needing a TRIGGER. (It also provides solutions for setting the default date and time.)